cpp/modmesh/buffer/pymod/wrap_SimpleArrayPlex.cpp

```diff
@@ -119,18 +119,18 @@ static void verify_python_value_datatype(pybind11::object const & value, DataTyp
         }
         break;
     }
-    case DataType::ComplexFloat32:
+    case DataType::Complex64:
     {
         if (!pybind11::isinstance<Complex<float>>(value))
         {
-            throw pybind11::type_error("Data type mismatch, expected complex float");
+            throw pybind11::type_error("Data type mismatch, expected Complex64");
```

Comment:
Using the data type name in error message.

Reply:
Nice.